src/mbgl/util/logging.cpp

```diff
@@ -43,8 +43,7 @@ void Log::record(EventSeverity severity, Event event, int64_t code) {
 }
 
 void Log::record(EventSeverity severity, Event event, int64_t code, const std::string &msg) {
-    if (currentObserver && severity != EventSeverity::Debug &&
```

Comment:
Was there a reason this was skipping `Debug` severity before?

Reply:
Not really; we aren't using Debug much. I changed the logging of one expected message from Debug to Info since it was a better fit anyway.